markdown/org/docs/patterns/charlie/instructions/en.md

```diff
@@ -215,78 +226,94 @@ Getting it just right is one of those things you'll end up cherishing each time
 </Tip>
 
 ## Step 10: Construct the fly
-
-### Close fly extension
+### Close the fly extension and serge
 
 Fold the fly extension on its fold line with good sides together.
 
 Now sew along the bottom, to close the fly extension.
 
 Trim back one half of the seam allowance to remove bulk before turning it good side out and pressing.
 
-### Serge the fly extension and fly facing
+#### Serge the fly extension and fly facing
 
 Serge (or zig-zag) along the open side of the fly extension. No need to serge the top as that will get caught in our waistband.
 
 while you're at it, also serge along the entirety of the fly facings J-shaped edge.
 
-### Cut away the extra seam allowance on the left leg front
+![Close Extension and Serge](CloseExtensionAndSerge.svg)
 
-There's extra seam allowance that runs along the fly. We only need this on the right leg panel.
-Cut it away on the left panel so you're left with the regular seam allowance only.
 
 ### Attach the fly facing
 
-Sew the fly facing to the left front. Place them with good sides together, aligning the top.
+Sew fly facing to left leg. You will be sewing slightly past the cross seam point, so be careful to keep the right leg seam allowance out of the way and aim to sew directly on top of the cross seam line.
 
-### Tuck in the tip of the fly facing
+![Attach Facing](AttachFacing.svg)
 
-Fold the bottom of the fly facing seam allowance inwards and secure with a few stitches.
+### Finish the fly facing
 
-### Edge stitch the fly facing to the seam alowance
+Trim the fly facing to half of the seam allowance.	
+Tuck in the tip of the fly facing and secure it with a few stitches.
+Edge stitch the fly facing to the seam allowance.
+Press the seam allowance toward the facing and understitch the facing for a clean finish. Again, keeping the right leg 	seam allowance out of the way.
 
-Sew the fly facing to the seam allowance. Make sure to leave the front panel out of it.
+![Finish Facing](FinishFacing.svg)
 
-### Press cross seam, fly facing and the fly edge
+### Edge stitch the zipper to the fly extension
 
-Press open the cross seam. Press the fly facing to the side.
+<tip>
 
-Then, fold over the extra seam allowance on the right front leg and press it down.
+**Zipper Length**
 
-### Baste the zipper to the fly extension
+It's a good idea to ensure your zipper stop ends before the indicated stitch line on the pattern. This will ensure you don't sew over your zipper stop and reduces tension on the bottom of the fly. If your zipper is too long, you can do a 0 length zig zag stitch to create a new stop (on just the zipper itself).
```

Comment:
```suggestion
It's a good idea to ensure your zipper stop ends before the indicated stitch line on the pattern. This will ensure you don't sew over your zipper stop and reduces tension on the bottom of the fly. If your zipper is too long, you can trim it down and create a new stopper by looping several stitches of thread inbetween the zipper teeth either by hand or using a 0 length zig zag stitch on your machine making sure to just sew on the zipper tape.
```

This may be more opinion. Either way maybe also adding a bit about using fray check or a dob of glue/superglue to secure the threads may be useful as well.